### ISSUE TITLE
2021 Contributors' Workshop: Install ember-a11y-testing

### ISSUE THREAD

**Issue by ijlee2** [ember-learn/upgrade-guide#71]
## Background

[ember-a11y-testing](https://github.com/ember-a11y/ember-a11y-testing) lets us check for accessibility issues using [axe](https://www.deque.com/axe/). While we cannot use `ember-a11y-testing` to check for every accessibility guideline, it does provide us a great minimum set of requirements to meet when we develop an app.

## Description

There are 3 steps to complete the issue.

### Step 1

Use Ember CLI to install `ember-a11y-testing`:

```bash
ember install ember-a11y-testing
```

Let's create and push a commit.

### Step 2

Next, let's find out how to run `ember-a11y-testing` in tests.

The `upgrade-guide` app has 2 routes, called `index` and `changes`. An ideal place to run an accessibility audit is in the application tests so that we check the entire page:

- `tests/acceptance/index-test.js`
- `tests/acceptance/changes-test.js`

For each test file, please add 1 test with the test description `Accessibility audit`. Here is the gist of the test:

```javascript
test('Accessibility audit', async function (assert) {
  // TODO: Visit the page
  // TODO: Do an accessibility audit

  assert.ok(true);
});
```

Note that `assert.ok(true);` is added because a test requires at least 1 assertion. The assertion can be called, if and only if, the accessibility audit has passed.

Run `yarn test` to check that all tests ... do not pass. We found an accessibility issue in the `changes` route template.

### Step 3

Use the error message to decide how to fix the accessibility issue.

<details>
<summary>Hint</summary>

The <code>&lt;CliUpdateCommands&gt;</code> component should not have added an <code>&lt;h3&gt;</code> element willy-nilly.

We can solve the accessibility issue by changing the tag to a <code>&lt;p&gt;</code>, for example.

</details>

## References

If you are stuck on writing the test, you can look at [a test for `ember-website`](https://github.com/ember-learn/ember-website/blob/b2af864c7704200e1beda0e9866d6650ea50bbd8/tests/acceptance/index-test.js#L26-L37). The particular example shows how you can make an exception, so that you can gradually improve accessibility in a large app.

**Comment by benjahur** [comment]
@ijlee2 I can take care of this :)

**Comment by ijlee2** [comment]
@benjahur Awesome, thank you for your help! If you have any questions, you can find me on Discord with the handle `@ijlee2`.

**Comment by ijlee2** [comment]
Completed in #72
